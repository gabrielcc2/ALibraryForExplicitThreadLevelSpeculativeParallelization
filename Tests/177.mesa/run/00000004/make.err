mesa4.c: In function ‘ReadMesh’:
mesa4.c:127:10: warning: ignoring return value of ‘fscanf’, declared with attribute warn_unused_result
mesa4.c:149:13: warning: ignoring return value of ‘fscanf’, declared with attribute warn_unused_result
mesa4.c: In function ‘SPECWriteIntermediateImage’:
mesa4.c:199:10: warning: ignoring return value of ‘fscanf’, declared with attribute warn_unused_result
