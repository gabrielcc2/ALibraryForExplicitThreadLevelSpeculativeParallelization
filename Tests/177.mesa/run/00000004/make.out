gcc -c -o accum.o            -O3    accum.c
gcc -c -o alpha.o            -O3    alpha.c
gcc -c -o alphabuf.o            -O3    alphabuf.c
gcc -c -o api1.o            -O3    api1.c
gcc -c -o api2.o            -O3    api2.c
gcc -c -o attrib.o            -O3    attrib.c
gcc -c -o bitmap.o            -O3    bitmap.c
gcc -c -o blend.o            -O3    blend.c
gcc -c -o clip.o            -O3    clip.c
gcc -c -o colortab.o            -O3    colortab.c
gcc -c -o context.o            -O3    context.c
gcc -c -o copypix.o            -O3    copypix.c
gcc -c -o depth.o            -O3    depth.c
gcc -c -o dlist.o            -O3    dlist.c
gcc -c -o drawpix.o            -O3    drawpix.c
gcc -c -o enable.o            -O3    enable.c
gcc -c -o eval.o            -O3    eval.c
gcc -c -o feedback.o            -O3    feedback.c
gcc -c -o fog.o            -O3    fog.c
gcc -c -o get.o            -O3    get.c
gcc -c -o hash.o            -O3    hash.c
gcc -c -o image.o            -O3    image.c
gcc -c -o light.o            -O3    light.c
gcc -c -o lines.o            -O3    lines.c
gcc -c -o logic.o            -O3    logic.c
gcc -c -o masking.o            -O3    masking.c
gcc -c -o matrix.o            -O3    matrix.c
gcc -c -o misc.o            -O3    misc.c
gcc -c -o mmath.o            -O3    mmath.c
gcc -c -o osmesa.o            -O3    osmesa.c
gcc -c -o pb.o            -O3    pb.c
gcc -c -o pixel.o            -O3    pixel.c
gcc -c -o pointers.o            -O3    pointers.c
gcc -c -o points.o            -O3    points.c
gcc -c -o polygon.o            -O3    polygon.c
gcc -c -o quads.o            -O3    quads.c
gcc -c -o rastpos.o            -O3    rastpos.c
gcc -c -o readpix.o            -O3    readpix.c
gcc -c -o rect.o            -O3    rect.c
gcc -c -o scissor.o            -O3    scissor.c
gcc -c -o shade.o            -O3    shade.c
gcc -c -o span.o            -O3    span.c
gcc -c -o stencil.o            -O3    stencil.c
gcc -c -o teximage.o            -O3    teximage.c
gcc -c -o texobj.o            -O3    texobj.c
gcc -c -o texstate.o            -O3    texstate.c
gcc -c -o texture.o            -O3    texture.c
gcc -c -o triangle.o            -O3    triangle.c
gcc -c -o varray.o            -O3    varray.c
gcc -c -o vb.o            -O3    vb.c
gcc -c -o vbfill.o            -O3    vbfill.c
gcc -c -o vbrender.o            -O3    vbrender.c
gcc -c -o vbxform.o            -O3    vbxform.c
gcc -c -o winpos.o            -O3    winpos.c
gcc -c -o xform.o            -O3    xform.c
gcc -c -o mesa4.o            -O3    mesa4.c
gcc     -O3    accum.o alpha.o alphabuf.o api1.o api2.o attrib.o bitmap.o blend.o clip.o colortab.o context.o copypix.o depth.o dlist.o drawpix.o enable.o eval.o feedback.o fog.o get.o hash.o image.o light.o lines.o logic.o masking.o matrix.o misc.o mmath.o osmesa.o pb.o pixel.o pointers.o points.o polygon.o quads.o rastpos.o readpix.o rect.o scissor.o shade.o span.o stencil.o teximage.o texobj.o texstate.o texture.o triangle.o varray.o vb.o vbfill.o vbrender.o vbxform.o winpos.o xform.o mesa4.o   -lm  -o mesa
